docs/security/host-certs.md

```diff
@@ -205,6 +205,56 @@ For instructions on how to request a host or service certificate using the Web
 interface please see the [user guide maintained by
 the OIM development team](https://confluence.grid.iu.edu/pages/viewpage.action?pageId=3244064).
 
+Requesting Host Certificate for HTCondor-CE Using Let's Encrypt
+---------------------------------------------------------------
+
+As an alternative to the above options, the Let's Encrypt software can be used to obtain host certificates.
+
+We use the following guide as a reference:
+
+   - https://github.com/cilogon/letsencrypt-certificates
+
+In particular, under Getting your host certificate, we follow the two "Setting up" sections.
+
+(See also: https://letsencrypt.org/getting-started/)
```

Comment:
The above lines should be reference material cited at the very bottom of the doc. If there isn't already a `Reference` section, create one

Reply:
Okie doke.